packages/storage/src/providers/AWSS3Provider.ts

```diff
@@ -446,7 +448,26 @@ export class AWSS3Provider implements StorageProvider {
 				throw error;
 			}
 		}
-
+		if (validateObjectExistence) {
+			const headObjectCommand = new HeadObjectCommand(params);
+			try {
```

Comment:
Nit (white-space makes it a little more readable)

```suggestion
			const headObjectCommand = new HeadObjectCommand(params);
			
			try {
```